src/injected/client-store-listener.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BaseClientStoresHub } from '../common/stores/base-client-stores-hub';
```

Comment:
should we import using relative paths here? so instead of
```typescript
import { BaseClientStoresHub } from '../common/stores/base-client-stores-hub';
```
we'll have
```typescript
import { BaseClientStoresHub } from 'common/stores/base-client-stores-hub';
```